charts/geoweb-frontend/README.md

```diff
@@ -99,11 +99,11 @@ kubectl delete namespace geoweb
 ```
 
 # Chart Configuration
-The following table lists the configurable parameters of the GeoWeb frontend chart and their default values.
+The following table lists the configurable parameters of the GeoWeb frontend chart and their default values specified in file values.yaml.
```

Comment:
Yet another tiny little thing to make it  all look oh so consistent:
can you please replace _values.yamls_ with _<yourvaluesfile>_.

Reply:
In this message it's referencing the values.yaml file found in the repository, in the other places its the values file created by the user.

Reply:
Ah right, didn't notice that. So everything seems to be all right!